.gitignore

```diff
@@ -1,10 +1,16 @@
+debug/
```

Comment:
Is this needed?

Reply:
that repo was 100% initiated with the proposed .gitignore generated by github, and I copied this from that.
I don't think it's actually needed because /target/debug is the actual location

Reply:
Not needed. The only thing `cargo new` puts in `.gitignore` is `/target`. I think it also used to add `**/*.rs.bak` and `Cargo.lock` for libraries, but no longer does it.